lib/src/window_manager.dart

```diff
@@ -399,11 +399,17 @@ class WindowManager {
   }
 
   /// Resizes the window to `width` and `height`.
-  Future<void> setSize(Size size, {animate = false}) async {
+  Future<void> setSize(
+    Size size, {
+    bool animate = false,
+    Offset? position,
```

Comment:
I don't think it's appropriate to add a position parameter to the setSize method, I'd improve setBounds so that it doesn't force the size and position to be provided